numpy/_core/_add_newdocs.py

```diff
@@ -1690,6 +1690,16 @@
     x : object
         A Python object that implements the ``__dlpack__`` and
         ``__dlpack_device__`` methods.
+    device : device, optional
+        Device on which to place the created array. Default: ``None``.
+        For Array-API interoperability only, so must be ``"cpu"`` if passed.
```

Comment:
```suggestion
        Must be ``"cpu"`` if passed which may allow importing an array
        that is not already CPU available.
```
In this case, it is unrelated to just array api compatibility.